Dockerfile

```diff
@@ -1,13 +1,16 @@
 # ================================
 # Build image
 # ================================
-FROM swift:5.2-bionic as build
-WORKDIR /build{{#fluent.db.is_sqlite}}
+FROM swift:5.2-focal as build
 
-# Install sqlite3
-RUN apt-get update -y \
-	&& apt-get install -y libsqlite3-dev \
-	&& rm -rf /var/lib/apt/lists/*{{/fluent.db.is_sqlite}}
+# Install OS updates and, if needed, sqlite3
+RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true \
+    && apt-get -q update \
```

Comment:
How much extra time does this tack onto image building? Shouldn't be expect that the swift image has done this recently?

Reply:
The combination of updating both the builder and runner images adds roughly 45 seconds for me on a mediocre Internet connection. Since the download occupies the bulk of the time taken, I would expect that time to drop as low as 15 seconds (combined!) on an actually _good_ connection. At worst it's no more than a minute unless a _huge_ number of packages are outdated, which would still barely be noticeable when MySQL or PostgreSQL is in use (thanks to the extra copies of BoringSSL).

While we should expect the Swift images to do this, that's only true inasmuch as the images themselves get updated. On average, the release-quality images only get updated by point releases, which often happens at long enough intervals to let a significant number of security fixes go by. New images are built if the Swift team finds something that's known to affect Swift, but we've seen multiple people raise concerns about vulnerability checkers sounding alerts about out of date packages in their Docker images - whether or not those packages affect Swift itself or the executables at runtime, the issues get listed anyway. I judged the small addition to build time to be worth getting the additional (if usually minor) security by default. In an ideal world it might even improve confidence in Vapor by removing the "lists of vulnerabilities show up on the dashboard" effect.

Reply:
Ok seems fair enough to me :+1: